packages/yoga/src/Button/web/Outline.jsx

```diff
@@ -1,57 +1,100 @@
 import React from 'react';
 import styled from 'styled-components';
-import { oneOf, bool } from 'prop-types';
+import { bool, node } from 'prop-types';
 import { hexToRgb } from '@gympass/yoga-common';
 
 import Button from './Button';
 
 const Outline = styled(Button)`
   ${({
     inverted,
-    variant,
     theme: {
       yoga: {
-        colors: { white, [variant]: color },
-        components: { button },
+        colors: { white },
+        components: {
+          button: {
+            types: { outline },
+          },
+        },
       },
     },
   }) => `
-    background-color: ${button.types.outline.backgroundColor.default};
-    border-color: ${color};
-    color: ${color};
+    background-color: ${outline.backgroundColor.default};
+    border: ${outline.border.width}px solid;
+    border-color: ${outline.font.default.color};
+    color: ${outline.font.default.color};
+
+    svg {
+      fill: ${outline.font.default.color};
+    }
 
     &:not([disabled]):hover, &:not([disabled]):focus {
-      background-color: ${hexToRgb(color, 0.25)};
-      box-shadow: none;
+      background-color: ${outline.backgroundColor.hover};
+      color: ${outline.font.hover.color};
+
+      svg {
+        fill: ${outline.font.hover.color};
+      }
     }
 
     &:not([disabled]):active {
-      background-color: ${button.types.outline.backgroundColor.pressed};
-      border-color: ${button.types.outline.font.pressed.color};
-      color: ${button.types.outline.font.pressed.color};
+      background-color: ${outline.backgroundColor.default};
```

Comment:
Maybe we have other properties overwriting to the same value, we could take a look at it.

Reply:
It's necessary because we are "extending" the "default" `Button` styles and there we have a different `background-color` when button is `:active`.

If we remove this line the background-color of the `Button.Outline:active` will be the same as the `Button:active`.